_posts/2016-10-31-recurse-center-week-5.md

```diff
@@ -0,0 +1,172 @@
+
+Sunday I went to one with a fellow RCer and it was great fun. I felt very nervous/excited at the start which made it hard to think. Can't wait to work through those feels by performing more.
+
+Definitely plan on doing more jams and meeting more improvisers in the city.
+
+#### **1-on-1 with a Random RCer**
+
+Last week a bunch of people signed up to do randomized 1-on-1s with other RCers. It was actually a wonderful experience.
+
+I had the joy of going for coffee with [@lushi](https://github.com/lushi) and [@yuliaju](https://github.com/yuliaju).
+
+I want to make sure this happens next batch as 1-on-1 time outside of RC is some of the best bonding time.
+
+### Intellectual ☕️
+
+#### **Vim**
+
+Last week I was chatting with an RCer and watching them coding with Emacs. That obviously got me sidetracked into changing text editors...to Vim!
+
+It made for a bit of a grueling week. I started by looking around at all the options:
+
+- Vanilla Vim?  
+- Neovim?  
+- MacVim?  
+- Vim + Janus?  
+- Vim + An open source config?  
+
+Along with each of those came the question of what plugins to install, what shortcuts to add, etc., etc., etc.
+
+Once I had everything installed, I then needed to dig into the configs so that all my favorite Atom functions worked (multi-cursor, linting, file-explorer, etc.).
+
+Once that was all done, it was time. I could finally...sit at my computer for a week struggling to master the keyboard.
+
+It's been a week though and I'm starting to feel frustrated in Atom. I think that's a good sign 😊.
+
+#### **EpubPress for 🔥🐺**
+
+I finally heard back from Firefox about EpubPress!
+
+It was such a back and forth:
+
+> "Hey, we need you to upload original source files."
+
+I did! I swear! Here's a zip and a link to the github repo. Dig around all you want.
+
+> "Can you please add instructions for building the source files?"
+
+Here are the instructions. But FYI they were also in that repo I sent you...
+
+> "Why am I seeing this error while trying to build?"
+
+Oh no! Because you are running on Windows. Linux plz.
+
+> "The built files don't match the files you submitted."
+
+The built files are all on one line. A single character could be different and the diffs would be off. Is it because I have different versions of the dependencies installed? Gahhh...
+
+Here's a build that isn't optimized.
+
+> "Please submit a build that isn't optimized."
+
+I just did...
+
+> "Oh...we'll look through it ASAP."
+
+KK thx 👍.
+
+So hopefully that gets resolved soon...
+
+#### **P5.js**
+
+Friday night I decided to open up [P5.js](https://p5js.org/) and give it a whirl.
+
+That quickly devolved into being fully sucked in. Over the course of the next day I started experimenting with randomly generated bubbles and easing functions.
+
+Here's a janky .gif of the result:
+
+![bubbles](/assets/posts/bubbles.gif){: .center-image }
+
+Spoke to some other P5 folks around RC and got some tips. Next project: Matrix Code!
```

Comment:
Rose said I should focus on Haskell because I'm more afraid of it. She's probably right :P. BUT AFTER THAT FO SHO!